activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -267,7 +267,7 @@ def at(secs)
     #
     #   Time.zone.now                 # => Fri, 31 Dec 1999 14:00:00 HST -10:00
     #   Time.zone.parse('22:30:00')   # => Fri, 31 Dec 1999 22:30:00 HST -10:00
-    def parse(str, now=now)
+    def parse(str, now=now())
```

Comment:
@tonyta thanks for the reference. I can only see he said about: `To get old behavior, try foo = foo().`